pkgs/top-level/haskell-packages.nix

```diff
@@ -161,11 +163,17 @@ in rec {
     };
     ghcjs = callPackage ../development/haskell-modules {
       ghc = compiler.ghcjs;
-      compilerConfig = callPackage ../development/haskell-modules/configuration-ghcjs.nix { };
+      compilerConfig =
+        extendCompose
+          (callPackage ../development/haskell-modules/configuration-ghcjs.nix { })
+          (callPackage ../development/haskell-modules/configuration-ghc-7.10.x.nix { });
```

Comment:
I noticed that `packageSetConfig` could be used, but I wasn't sure it was intended for this and `compilerConfig` sounds more fitting than `packageSetConfig`. Alternatively we could introduce a new argument `compilerBackendConfig` or make the `*Config` arguments take lists of configs. Anyway I'm gonna update the PR according to your suggestion.